packages/swing-store/src/snapStore.js

```diff
@@ -88,8 +88,8 @@ export const fsStreamReady = stream =>
  *   tmpName: typeof import('tmp').tmpName,
  *   unlink: typeof import('fs').promises.unlink,
  * }} io
- * @param {object} [options]
- * @param {boolean | undefined} [options.keepSnapshots]
+ * @param {object} options
```

Comment:
That seems like a regression, and I've also been hit by this in my own code. There is an explicit init of the options param with `= {}` so it is actually optional. This really feels like a TS bug. I need to read more on why this is happening.

Reply:
Apparently there is already a bug filled on this: https://github.com/microsoft/TypeScript/issues/50286